lib/spinor_noise.cu

```diff
@@ -226,9 +133,7 @@ namespace quda {
   void spinorNoise(ColorSpinorField &src, unsigned long long seed, QudaNoiseType type)
   {
     RNG *randstates = new RNG(src, seed);
```

Comment:
Can be postponed for after GK but with Init/Release gone this really should randstates really should not be a ptr.
`RNG randstates(src,seed);` should do.

Reply:
Fixed in 92f2c2e0b (and also gauge_random.cu as well)